theano/tensor/tests/test_opt.py

```diff
@@ -6088,7 +6091,9 @@ def test_local_useless_reshape():
     f = theano.function([i], m, mode=mode)
     topo = f.maker.fgraph.toposort()
     assert not any(isinstance(n.op, tensor.basic.Reshape) for n in topo)
-
+    theano.printing.debugprint(f, print_type=True)
```

Comment:
You can remove that one, we try to avoid printing things during tests now (even though many tests still do).